tests/e2e/assets/webpack/test-server-app/app/injectable.ts

```diff
@@ -4,5 +4,5 @@ import {DOCUMENT} from '@angular/platform-browser';
 
 @Injectable()
 export class MyInjectable {
-  constructor(public viewContainer: ViewContainerRef, @Inject(DOCUMENT) public doc) {}
```

Comment:
This is to test the downgrading of a non-annotated Angular class. I think the test needs to happen, I don't care much with which class.  Does this test makes your code fail? How so?